frontend/wallet/src/render/CodaProcess.re

```diff
@@ -59,17 +59,20 @@ let useHook = () => {
   let (_state, dispatch) =
     Hooks.Reducer.useReducer(reduce, {State.args: [], mode: Stable});
 
-  // let () ÷=
-  // React.useEffect0(() => {
-  // let token = MainCommunication.listen();
-  // let args = None;
-  //   switch (getLocalStorageNetwork()) {
-  //   | "" => []
-  //   | s => ["-peer", s]
-  //   };
-  // dispatch(Action.StartCoda(args));
-  // Some(() => MainCommunication.stopListening(token));
-  // });
-
   dispatch;
 };
+
+let useStartEffect = dispatch => {
+  React.useEffect0(() => {
+    let token = MainCommunication.listen();
+    let args =
+      switch (getLocalStorageNetwork()) {
+      | "" => []
+      | s => ["-peer", s]
```

Comment:
I realize this is just uncommented but the behavior here with getLocalStorageNetwork, defaultNetwork and defaultPeers seems a little funny/wrong

Reply:
Take a look now, changed to use defaultPeers.